packages/layout-grid/Grid.js

```diff
@@ -0,0 +1,37 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Grid = (props) => {
+  const {
+    align,
+    children,
+    className,
+    fixedColumnWidth,
+    ...otherProps
+  } = props;
+
+  const classes = classnames('mdc-layout-grid', className, {
+    [`mdc-layout-grid--align-${align}`]: !!align,
+    'mdc-layout-grid--fixed-column-width': fixedColumnWidth,
+  });
+
+  return (
+    <div className={classes} {...otherProps}>{children}</div>
```

Comment:
div should be configurable. See https://github.com/material-components/material-components-web-react/pull/327/files#diff-a4ae7491ba4f26fa221dde04643b6fa0R7